packages/react/types/css.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { CssFunction } from '@compiled/react';
+
+declare module 'react' {
```

Comment:
To address error such as 
`Property 'css' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>`

Not sure if `CssFunction` is the most appropriate type, let me know if there is better one!

Reply:
We need to think about how we're going do this, as this will break jsx runtime overrides

Reply:
Raised https://github.com/atlassian-labs/compiled/issues/1323